platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/item/IWorkflowableObject.java

```diff
@@ -34,6 +34,22 @@ default boolean isWorkflowStarted() {
 		return getWorkflowItem() != null;
 	}
 
+	/**
+	 * Executes the longest possible cascade of auto transitions for the
+	 * {@link AGWorkflowItem}.
+	 * <p>
+	 * If this {@link IWorkflowableObject} is not associated with a
+	 * {@link AGWorkflowItem}, nothing will happen.
+	 *
+	 * @see #getWorkflowItem()
+	 */
+	default void executeCascadingAutoTransitions() {
```

Comment:
same though concerning the name as in `AGWorkflowItem`

Reply:
see above